db/db_test.cc

```diff
@@ -4292,6 +4292,67 @@ TEST_F(DBTest, DynamicCompactionOptions) {
 }
 #endif  // ROCKSDB_LITE
 
+// Test dynamic compaction_options_fifo
+TEST_F(DBTest, DynamicFIFOCompactionOptions) {
+  Options options;
+  options.create_if_missing = true;
+  DestroyAndReopen(options);
+
+  ASSERT_OK(dbfull()->SetOptions({{"compaction_options_fifo", "11"}}));
```

Comment:
I feel this usage is confusing. Is it really a good idea to allow saying "compaction_options_fifo = 11" without specifying which field to set?